### PR TITLE
feat: center-align message via boxen@0.6.0

### PR DESCRIPTION
**Pull request by nexdrew**
Replace the explicit use of `ansi-align` with the new `align: 'center'` option [available in `boxen@0.6.0`](https://github.com/sindresorhus/boxen/tree/v0.6.0#align).

Add center-alignment for the default `.notify()` message, with test to verify output.

The "update check failed" message used when `configstore` throws was already center-aligned, but this PR will defer alignment to `boxen`.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -2,9 +2,17 @@
...
 var fs = require('fs');
+var util = require('util');
 var clearRequire = require('clear-require');
+var FixtureStdout = require('fixture-stdout');
+var stripAnsi = require('strip-ansi');
 var updateNotifier = require('./');
 
+function clearRequireAnd(modules, fn) {
+	modules.forEach(clearRequire);
+	fn();
```
Comment (SBoudrias) [inline test.js]: This is synchronous. Why provide a callback?

Comment (nexdrew) [inline test.js]: I don't know, seemed like a good idea at the time, but I guess it doesn't really help make things less verbose. 😅 I can change it.

Comment (SBoudrias) [inline test.js]: Awesome, the rest of the code looks good to me.

**Comment by nexdrew** [comment]
@SBoudrias Removed the confusing "synchronous `clearRequire()` + callback" function with commit 277df2b. Let me know if you see anything else.

**Comment by SBoudrias** [comment]
Cool, thanks for sending this out!

**Comment by nexdrew** [comment]
@SBoudrias No problem, thanks for the review and merge!

**Pull request merged**